modules/vscode-extensions/vscode-lsp/client/src/extension.ts

```diff
@@ -16,12 +16,9 @@ import {
 let client: LanguageClient;
 import * as fs from 'fs';
 import * as Net from 'net';
-const xmlQuery = require('xml-query');
-const XmlReader = require('xml-reader');
-var xmlFilePath;
 import { FileHandler } from './lspModules/fileHandler';
 import { PreviewManager } from './lspModules/PreviewManager';
-import { Wso2OAuth } from './lspModules/oAuthService';
+var xmlFilePath;

```

Comment:
why this was XML?

Reply:
Its to keep the file path of the service providers xml